serialization_spec/plugins.py

```diff
@@ -63,7 +63,7 @@ def transform(self, value):
 class MethodCall(SerializationSpecPlugin):
     def __init__(self, name, required_fields=None):
         self.name = name
-        self.required_fields = set(required_fields) or {}
+        self.required_fields = set(required_fields) if required_fields else set()
```

Comment:
I guess I meant this 😄 

```suggestion
        self.required_fields = set(required_fields or {})
```

Reply:
The current way is nice and explicit, and has fewer unnecessary type cases.

Reply:
BUT.... MOAR CHARACTERS ;)